tests/test_json_regex.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+from json import loads, dumps
+from apluslms_roman.utils.path_mapping import json_re, env_value_to_dict
+
+test_case_loadable = (
+    True,
+    False,
+    None,
+    123,
+    -123,
+    3.14,
+    -3.14,
+    {'foo': 'bar'},
+    [1, 2, 3],
+    "foo bar"
+)
+
+test_case_not_loadable = (
+    "/foobar.py",
+    "text",
+    "yes",
+    "0123123",
+)
+
+
+def try_loads_json(json_str):
+    try:
+        data = loads(json_str)
+    except ValueError:
+        data = None
+    return data
+
+
+class TestJsonRegex(unittest.TestCase):
+
+    def test_loadable_match(self):
+        for i, case in enumerate(test_case_loadable):
+            with self.subTest(i=i):
+                self.assertTrue(json_re.match(dumps(case))is not None, msg="Testing:{}".format(case))
+
+    def test_not_loadable_match(self):
+        for i, case in enumerate(test_case_not_loadable):
+            with self.subTest(i=i):
+                self.assertFalse(json_re.match(case) is not None, msg="Testing:{}".format(case))
+
+    def test_loadable_equal(self):
+        for i, case in enumerate(test_case_not_loadable):
+            with self.subTest(i=i):
+                self.assertEqual(try_loads_json(dumps(case)), case, msg="Testing:{}".format(case))
+
+    def test_not_loadable_equal(self):
+        for i, case in enumerate(test_case_not_loadable):
+            with self.subTest(i=i):
+                self.assertNotEqual(try_loads_json(dumps(case)), -1, msg="Testing:{}".format(case))
+
+    def test_loadable_type(self):
+        for i, case in enumerate(test_case_loadable):
+            with self.subTest(i=i):
+                self.assertNotEqual(env_value_to_dict(dumps(case)), dumps(case), msg="Testing:{}".format(case))
```

Comment:
Use different classes for different functions/features, so it's easier to notice when reading the file.